### PR TITLE
valgrind: added version 3.15.0

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

**Comment by mgeplf** [comment]
Release documented here:
http://valgrind.org/downloads/current.html

File: var/spack/repos/builtin/packages/valgrind/package.py [hunk 1]
```diff
@@ -22,6 +22,7 @@ class Valgrind(AutotoolsPackage):
     git      = "git://sourceware.org/git/valgrind.git"
 
     version('develop', branch='master')
+    version('3.15.0', '46e5fbdcbc3502a5976a317a0860a975')
```
Comment (alalazo) [inline var/spack/repos/builtin/packages/valgrind/package.py]:
```suggestion
    version('3.15.0', sha256='417c7a9da8f60dd05698b3a7bc6002e4ef996f14c13f0ff96679a16873e78ab1')
```
Obtained with the command:
```console
$ spack checksum valgrind 3.15.0
==> Found 1 version of valgrind:
  
  3.15.0  https://sourceware.org/pub/valgrind/valgrind-3.15.0.tar.bz2

==> How many would you like to checksum? (default is 1, q to abort) 
==> Downloading...
==> Fetching https://sourceware.org/pub/valgrind/valgrind-3.15.0.tar.bz2
########################################################################################################################################### 100,0%
==> Checksummed 1 version of valgrind

    version('3.15.0', sha256='417c7a9da8f60dd05698b3a7bc6002e4ef996f14c13f0ff96679a16873e78ab1')
```

**Comment by alalazo** [comment]
Thanks, just waiting for Travis to merge this!

**Comment by mgeplf** [comment]
No problem, my pleasure.

**Pull request merged**